### ISSUE TITLE
unbound variable

### ISSUE THREAD

**Issue by MeikelLP** [firefly-iii/firefly-iii#3062]
**Bug description**
I am running Firefly III version `5.0.1`, and my problem is:

Docker image is broken unless you explicity set the debug variables:

```
/entrypoint.sh: line 88: DKR_CHECK_SQLITE: unbound variable
``` 

**Steps to reproduce**
Only in `5.0.1`. Not present in `5.0.0`

**Bonus points**

- [x] Nobody reported this bug before
- [x] I have added a stack trace from my log files.

**Comment by tgkenney** [comment]
I'm having the same issue here with my container deployment.  I noticed in the docs there seems to be a new env variable `DB_CONNECTION` that I don't believe was required before.  I have tried setting this but it did not solve the issue

**Comment by JC5** [comment]
A new image has been built for 5.0.1 that should solve this issue. Thanks for reporting this. Version 5.0.3 is about to come out which should also not have this issue.

@tgkenney The environment variable `DB_CONNECTION` has been around since 2014, but this variable could be related to a bug in another part of the installation script, #3063, which has been fixed in upcoming version 5.0.3.
